styles/components/cards.css

```diff
@@ -0,0 +1,156 @@
+.card {
+  display: flex;
+  border: 1px solid var(--inactive-color);
+  flex-direction: column;
+  margin: 1rem 0.5rem;
+  max-height: 30%;
+  max-width: 360px;
```

Comment:
try to use rem units  for responsiveness.

Reply:
Done..!!